### PR TITLE
Send Scene: Refactor the design Flip Input Modal component

### PR DESCRIPTION
**Pull request by amanalili**
#### PR Requirements

Note: Remove Modal Close Button. as the Done Button or the check button on the android keyboard does the same thing

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

Screen Shot 2021-03-11 at 7 10 55 PM

### CODE REVIEW

File: src/components/modals/FlipInputModal.js [hunk 1]
```diff
@@ -92,6 +91,19 @@ class FlipInputModalComponent extends React.PureComponent<Props, State> {
 
   handleSendMaxAmount = () => this.props.updateMaxSpend(this.props.walletId, this.props.currencyCode)
 
+  renderTitle = () => {
+    const styles = getStyles(this.props.theme)
+    return (
+      <View style={styles.headerContainer}>
```
Comment (swansontec) [inline src/components/modals/FlipInputModal.js:96]: You should try using `justifyContent: "space-between"` on this container component, so you can get rid of the `styles.spacer` view.

File: src/components/modals/FlipInputModal.js [hunk 2]
```diff
@@ -164,12 +176,13 @@ class FlipInputModalComponent extends React.PureComponent<Props, State> {
...
         <ThemedModal bridge={this.props.bridge} onCancel={this.handleCloseModal}>
           <ScrollView style={{ maxHeight: this.props.theme.rem(23) }} keyboardShouldPersistTaps="handled">
-            <ModalTitle>{s.strings.string_enter_amount}</ModalTitle>
+            {this.renderTitle()}
             {this.renderExchangeRates()}
             {this.renderBalance()}
             {this.renderFlipInput()}
             {this.renderFees()}
             <PrimaryButton label={s.strings.send_confirmation_max_button_title} marginRem={1} onPress={this.handleSendMaxAmount} />
+            <ModalCloseArrow onPress={this.handleCloseModal} />
```
Comment (swansontec) [inline src/components/modals/FlipInputModal.js]: Wait, isn't this supposed to go away, according to the description?

File: src/components/services/Services.js [hunk 3]
```diff
@@ -73,6 +73,7 @@ export class Services extends React.PureComponent<Props> {
     return (
       <MenuProvider>
         <Main />
+        <Airship avoidAndroidKeyboard statusBarTranslucent />
```
Comment (swansontec) [inline src/components/services/Services.js:76]: Oh, yeah, good fix! This makes sense.

**Review by swansontec — APPROVED** [review]
It looks like you could just squash the last commit into the first commit, since you add a close arrow & then remove it at the end.

You could also eliminate one view using a different CSS style, but I won't hold up the release over these two little issues. You can fix them if you feel like that's worth doing.

**Comment by amanalili** [comment]
updated

**Pull request merged**